### PR TITLE
Optimizer Code for Wavax and DaiE

### PR DESCRIPTION
**Pull request by jomarip**
Added the Controller to the Tests and Helper. Also new snowglobes

### CODE REVIEW

File: contracts/strategies/aave/strategy-aave-dai.sol [hunk 1]
```diff
@@ -13,7 +13,7 @@ import "../../interfaces/aave.sol";
 import "../strategy-base.sol";
 import "../strategy-joe-farm-base.sol";
 
-contract StrategyAaveDai is StrategyBase, Exponential {
+contract StrategyAaveDaiE is StrategyBase, Exponential {
```
Comment (fuzzylemma) [inline contracts/strategies/aave/strategy-aave-dai.sol:16]: for the `aave-daie` snowglobe, a new file was introduced with an 'e' suffix. 

Is this a renaming of AaveDai to AaveDaiE snowglobe and strategy or new ones?  does it make sense to keep `contracts/snowglobes/aave/snowglobe-aave-dai.sol` ?

Comment (zeroecksYeti) [reply contracts/strategies/aave/strategy-aave-dai.sol:16]: If we're renaming the `aave-dai`contract to `aave-daie` and deprecating the `aave-dai` pool, we should remove  the old snowglobe as a new one (aave-daie) is included in this branch

File: test/FoldingStrats/benqi.ts [hunk 2]
```diff
@@ -31,7 +31,7 @@ const tests = [
     // },
     {
         name: "BenqiWavax",
-        slot: 1,
+        slot: 3,
```
Comment (zeroecksYeti) [inline test/FoldingStrats/benqi.ts:34]: I don't know if this is correct, has it been tested @jomarip ?

**Review by AbigailTCameron — APPROVED** [review]
LGTM!

**Review by AbigailTCameron — APPROVED** [review]
LGTM

**Pull request merged**